circuit/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "wormhole-circuit"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+description.workspace = true
+license.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+plonky2.workspace = true
+poseidon-resonance.workspace = true
+hex.workspace = true
```

Comment:
I think `hex` is used exclusively within the circuit, so no need to have it as a workspace dependency.